app/components/Hotkeys.vue.ts

```diff
@@ -10,21 +10,19 @@ import HotkeyGroup from './HotkeyGroup';
   components: { HotkeyGroup },
 })
 export default class Hotkeys extends Vue {
-  @Inject()
-  sourcesService: SourcesService;
+  @Inject() private sourcesService: SourcesService;
+  @Inject() private scenesService: ScenesService;
+  @Inject() private hotkeysService: HotkeysService;
 
-  @Inject()
-  scenesService: ScenesService;
+  hotkeySet: IHotkeysSet = null;
 
-  @Inject()
-  hotkeysService: HotkeysService;
-
-  hotkeySet = this.hotkeysService.getHotkeysSet();
-
-  created() {
+  mounted() {
     // We don't want hotkeys registering while trying to bind.
     // We may change our minds on this in the future.
     this.hotkeysService.unregisterAll();
+
+    // Render a blank page before doing synchronous IPC
+    setTimeout(() => (this.hotkeySet = this.hotkeysService.getHotkeysSet()), 100);
```

Comment:
Our ChildWindow component doesn't handle this?

Reply:
Unfortunately not, because the top level component (`Settings.vue`) is already mounted and usable.  This is for when you switch to the `Hotkeys` tab, and things hang for a little bit.